ot3_state_manager/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml we can fold this setting into pyproject.toml

Reply:
https://github.com/Opentrons/ot3-firmware/pull/418/commits/7d71df09720e545a05b7d9d7ef1b08717593ec2f